modules/tlc2/overrides/SequencesExt.java

```diff
@@ -266,11 +268,20 @@ public static Value replaceAllSubSeq(final Tool tool, final ExprOrOpArgNode[] ar
 		final IValue t = tool.eval(args[2], c, s0, s1, control, cm);
 		
 		if (r instanceof StringValue && s instanceof StringValue && t instanceof StringValue) {
-			final String st = ((StringValue) t).toUnquotedString();
-			final String ss = ((StringValue) s).toUnquotedString();
-			final String sr = ((StringValue) r).toUnquotedString();
+			final String st = ((StringValue) t).getVal().toString();
+			final String ss = ((StringValue) s).getVal().toString();
+			final String sr = ((StringValue) r).getVal().toString();
+			
+			if(ss.equals("")) {
+				StringBuilder result = new StringBuilder(sr);
+				for(int i=0;i<st.length();i++) {
+					if(i != 0) result.append(sr);
```

Comment:
Add `{}`?

Reply:
Missed that one 😅 Done, thanks for the review!